web/source/kmwnative.js

```diff
@@ -94,22 +94,30 @@
   * 
   * @param  {Object}  Ln  keyboard stub object    
   */    
-  keymanweb.keyboardUnavailable = function(Ln)
-  {
-    return window.setTimeout(function() {
-        util.wait(false);
-        var Ps=keymanweb._KeyboardStubs[Ln],kbdName=Ps['KN'],lgName=Ps['KL'];
-        kbdName=kbdName.replace(/\s*keyboard\s*/i,'');
+  keymanweb.keyboardUnavailable = function(Ln) {
+    keymanweb.loadFailureHandler = function(altString) {
```

Comment:
Yes, I've added one new, visible name within the `keymanweb` object - by making this a visible method, we can reuse the original error callback while allowing customization for other errors if detected via script insertion events.

The method is used both here and in `keymanweb.installKeyboard`, hence the need for that visible name.